spyder_kernels/utils/iofuncs.py

```diff
@@ -521,6 +522,72 @@ def save_auto(data, filename):
     pass
 
 
+class WriteContext(object):
```

Comment:
Let's move the two classes you added here to `comms/utils.py` because this module is used to load and save different types of data. Also, I think they are more related to comms than to other functionality in Spyder-kernels.